docs/design/software-design-specification.adoc

```diff
@@ -211,21 +211,19 @@ Input parameters: TODO
 
 | SDS-TASK-9
 | `start` binary
-a| The task checks out the repository of a given URL and Git ref into the mounted workspace, cleaning previous contents, except for the caching area at `./ods-cache`. If the checked out `ods.y(a)ml` configures any child repositories, those are checked out as well from the configured URL and Git ref. If a release branch (`release/<VERSION>`) corresponding to the current version exists, it is preferred. All checkouts are shallow and include submodules.
+a| The task checks out the repository of a given URL and Git ref into the mounted workspace, cleaning previous contents, except for the caching area at `./ods-cache`. If the checked out `ods.y(a)ml` configures any child repositories, those are checked out as well from the configured URL and Git ref. If a release branch (`release/<VERSION>`) corresponding to the current version exists, it is preferred. All checkouts are by default shallow (depth=1) and include submodules.
```

Comment:
> All checkouts are by default shallow (depth=1) and include submodules.
Has this changed?

Reply:
No it hasn't -  I just added clarifying notes.

What has "changed" is that now the depth is just a default because with trigger-based params you are able to overwrite it!